src/test/login-test.ts

```diff
@@ -0,0 +1,151 @@
+import axios, { AxiosResponse } from 'axios';
+import bcrypt from 'bcrypt';
+import { expect } from 'chai';
+import { serverUrl } from '../setup-server';
+import { AppDataSource } from '../data-source.js';
+import { User } from '../entity/User.js';
+
+interface CreateUserInputData {
+  name: string;
+  email: string;
+  birthDate: string;
+  password: string;
+}
+
+interface LoginInputData {
+  email: string;
+  password: string;
+}
+
+async function postLogin(loginInputData: LoginInputData): Promise<AxiosResponse> {
+  const response = await axios.post(`${serverUrl}graphql`, {
+    query: `
+    mutation Login($data: LoginInput!) {
+      login(data: $data) {
+        user {
+          birthDate
+          email
+          id
+          name
+        }
+        token
+      }
+    }
+      `,
+    variables: {
+      data: loginInputData,
+    },
+  });
+
+  return response;
+}
+
+describe('login mutation', () => {
+  beforeEach(async () => {
+    const userRepository = AppDataSource.getRepository(User);
+    await userRepository.clear();
+  });
+
+  it('should be able to login', async () => {
+    const inputUser: CreateUserInputData = {
+      name: 'John Phill',
+      email: 'john@example.com',
+      password: 'password123',
+      birthDate: '1990-01-01',
+    };
+
+    const inpuLogin: LoginInputData = {
+      email: inputUser.email,
+      password: inputUser.password,
+    };
+
+    const inputUserWithoutPassword = {
+      name: inputUser.name,
+      email: inputUser.email,
+      birthDate: inputUser.birthDate,
+    };
+
+    const expectedResponseWithoutPassword = {
+      user: inputUserWithoutPassword,
+      token: '',
+    };
```

Comment:
```suggestion
    const expectedResponseWithoutPassword = {
      user: { ...inputLogin, birthDate: inputUser.birthDate, },
      token: '',
    };
```

Um setup mais limpo!